st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -348,35 +426,35 @@ def encrypt_archive(archive_file_path, debug=False):
     encrypted_archive_output_file_path = archive_file_path + '.asc'
```

Comment:
Hi @andrew-regan , as of now we have used the existing GPG_KEY for testing.

We have issue on defining the GPG_KEY in yaml file, digging into that. As a work around to check the wrapper script we have defined the GPG_KEY in a file(/tmp/gpg_key.txt) and gave the path to the yaml file.

Please correct me here if we are wrong.